README.md

```diff
@@ -35,3 +35,12 @@ Make sure the service is up first using `docker-compose up` then run:
 ```sh
 docker-compose exec core python manage.py test
 ```
+
+to run unit tests. Or, to load cancer-static data:
```

Comment:
What unit tests?

Can you place the data loading under a separate header than "Running tests locally"? You can use pound symbols ## to set the header level.